.github/workflows/unit-test.yml

```
@@ -1,44 +1,43 @@
-name: Unit Test
+# name: Unit Test
```

actionlint (1.7.7)

workflow is empty

(syntax-check)